### DISCUSSION TITLE
If a docstring of a pydantic model has multiple paragraphs, every paragraph past the first are treated as "preformatted text"

### DISCUSSION THREAD

**Discussion by exhuma** [fastapi/fastapi#10119] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from pydantic import BaseModel


class PayloadOne(BaseModel):
    """
    This is an example docstring with leading whitespace

    The whole block is treated as "preformatted text" due to the leading spaces.
    """
    example: str

class PayloadTwo(BaseModel):
    """This is an example docstring with no leading whitespace

    The second (this) block is treated as "preformatted text" due to the leading
    spaces.
    """
    example: str

APP = FastAPI(title="Bristow Flight Logs")

@APP.get("/")
def index(foo: PayloadOne, bar: PayloadTwo):
    return "OK"
```

### Description

Following [PEP-257#handling-docstring-indentation](https://peps.python.org/pep-0257/#handling-docstring-indentation), the following two docstrings should be identical:

```python
def foo():
    """A multi-line
    docstring.
    """

def bar():
    """
    A multi-line
    docstring.
    """
```

The example code above uses both formats.

When a pydantic model is documented like that, and FastAPI uses those docs to generate the `swagger` file, the output is badly indented. This causes blocks to be displayed as "pre-formatted" code instead of simple text. Here is an example screenshot that highlights the issue of the code above.

![Sample](https://github.com/tiangolo/fastapi/assets/65717/55883f26-a08b-4ec8-b2cb-0b4f58caecc3)

### Operating System

Linux, Windows

### FastAPI Version

0.100.0

### Pydantic Version

2.0.2

### Python Version

3.11.4

**Comment by exhuma** [comment]
I just noticed that this is also visible in the pydantic documentation so it seems to me that this is a pydandic issue after all. I did not spot that yesterday. Can this be closed then? Or should we keep it open to keep track of it?

I opened a corresponding ticket so for me it's fine to just close this: https://github.com/pydantic/pydantic/issues/7195

**Reply by Kludex** [reply]
Let's continue on Pydantic. 👍 

Thanks. 🙏
